packages/components/src/menu-group/index.js

```diff
@@ -19,6 +19,9 @@ export function MenuGroup( {
 	className = '',
 	instanceId,
 	label,
+	role = 'menu',
```

Comment:
It feels wrong to see the default role set `menu` for the component which is called `MenuGroup` and then see it overridden back to `group` in other places. It seems like we need to introduce another component called `Menu` which will have `role` set to `menu`, its class name which will hide the label for the menu, and will always have `useEventToOffset` set to `true`. This will make it possible to leave `MenuGroup` usage unchanged.

Reply:
@gziolo should we spin up a new issue for that?

Reply:
I would feel comfortable doing it as part of this PR because in the current form you might see unexpected `role="menu"` in all plugins using `MenuGroup`.